packages/fxa-admin-panel/public/index.html

```diff
@@ -11,7 +11,7 @@
       name="viewport"
       content="width=device-width,initial-scale=1,minimum-scale=1,maximum-scale=2,user-scalable=yes"
     />
-    <link rel="stylesheet" href="styles/main.css" />
```

Comment:
Do we not need this `rel` attribute?

Reply:
As far as I can tell, no, most browsers should download and handle it just fine. I removed it because were encountering [this Express issue](https://stackoverflow.com/q/44657829/717633). I'm definitely open to alternatives, this was just a quick and dirty solution I found.